11-query/05-updating-data.md

```diff
@@ -112,46 +115,46 @@ GraqlInsert insert_query = Graql.match(
 [tab:end]
 </div>
 
-However, to modify or delete role players, the current limitations require us to [delete the instances of the relation](../11-query/04-delete-query.md#delete-instances-of-a-relation-type) with the current role players and [insert the new instance of the relation](../11-query/03-insert-query.md#insert-instances-of-a-relation-type) with the new roleplayers.
+To replace a role player, we combine extending the relation with removing a role player:
 
 <div class="tabs dark">
 
 [tab:Graql]
 ```graql
-## inserting the new
 match
-  $p isa person, has name "Amabo";
-  $org isa organisation, has name "Etihw Esouh";
-insert $emp (employer: $org, $employee: $p) isa employment;
+  $org isa organisation, has name "Pharos";
+  $new-org isa organisation, has name "Medicely";
+  $emp (employer: $org, employee: $p) isa employment;
+insert
+  $emp (employer: $new-org);
 
-## deleting the old
 match
-  $p isa person, has name "Prumt";
-  $org isa organisation, has name "Etihw Esouh";
-  $emp (employer: $org, employee: $p) isa employment;
-delete $emp;
+  $emp (employer: $org, employer: $new-org, employee: $p) isa employment;
+  $org isa organisation, has name "Pharos";
+  $new-org isa organisation, has name "Medicely";
+delete
+  $emp (employer: $org);
 ```
 [tab:end]
 
 [tab:Java]
 ```java
-GraqlInsert insert_query = Graql.match(
-  var("p").isa("person").has("name", "Amabo"),
-  var("org").isa("organisation").has("name", "Wieth Souhe")
-).insert(
-  var("emp").isa("employment").rel("employer", var("org")).rel("employee", var("p"))
-);
-
-GraqlDelete delete_query = Graql.match(
-  var("p").isa("person").has("name", "Prumt"),
-  var("org").isa("organisation").has("name", "Wieth Souhe"),
-  var("emp").isa("employment").rel("employer", var("org")).rel("employee", var("p"))
-).delete("emp");
+GraqlInsert query = Graql.match(
+  var("org").isa("organisation").has("name", "Pharos"),
+  var("new-org").isa("organisation").has("name", "Medicely"),
+  var("emp").isa("employment").rel("employer", "org").rel("employee", "p")
+).insert(var("emp").rel("employer", "new-org"));
+
+GraqlDelete deleteQuery = Graql.match(
+  var("org").isa("organisation").has("name", "Pharos"),
+  var("new-org").isa("organisation").has("name", "Medicely"),
+  var("emp").isa("employment").rel("employer", "org").rel("employer", "new-org").rel("employee", "p")
+).delete(var("emp").rel("employer", "org"));
 ```
 [tab:end]
 </div>
 
-This query updates the `employee` roleplayer of the `employment` relation where the `employer` is an `organisation` named `"Wieth Souhe"`.
+After these queries, all employments by the organisation named `Pharos` were replaced with the organisation name `Medicely`.
```

Comment:
...with the organisation name**d** `Medicely`

Reply:
thanks!